common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
In the LMS it works fine on the Teams page which doesn't use JQuery UI. The bundle shows that it only includes the extra pieces:

http://clytwynec.sandbox.edx.org/static/teams/js/teams_tab_factory.3132081303d2.js?raw

    /*! jQuery UI - v1.11.4 - 2015-09-10
    * http://jqueryui.com
    * Includes: core.js, widget.js, mouse.js, selectable.js
    * Copyright 2015 jQuery Foundation and other contributors; Licensed MIT */

The new bundle add 18KB to the size of the page, so that seems reasonable.